change/@fluentui-react-carousel-preview-533b9b0e-1769-4408-b096-4984d68abfaa.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32154&commitId=2d234b4c1b83ddbd87a2ee6e0ec175c88274453f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>